peps/pep-0705.rst

```diff
@@ -363,6 +365,35 @@ Notes:
     b: B = { "x": 3 }
     c: C = a | b  # Accepted by type checker: "x" will always come from b
 
+Minimum support
+^^^^^^^^^^^^^^^
+
+A type checker may reject sound assignments, for instance if it is unable to support bidirectional type inference. This section provides a minimum standard of support that a type checker should provide, so users can write code that will be consistently accepted across all type checkers.
+
+Type checkers may reject any code not conforming to this model, but the error should be explicit that this is a type checker limitation, and that the user code is not necessarily unsound. It may be reasonable to use a distinct code for suppressing such issues, so if the checker later supports full inference and the code is indeed unsound, the issue will no longer be suppressed.
+
+A type checker should always permit ``a | b`` if the following holds:
+
+* Every key in ``A`` is also in ``B``
+* Every non-required key in ``B`` is also in ``A``.
+
+A type checker may treat ``a | b`` as returning a fixed TypedDict ``F`` constructed by the following rules:
+
+* The key set of ``F`` is the union of the key sets of ``A`` and ``B``
+* All items of ``F`` are non-read-only
+* A key in ``F`` is required if and only if it is required in either ``A`` or ``B``
+* If a key is required in ``B``, its value type in ``F`` is the same as in ``B``
+* If a key is not required in ``B``, its value type in ``F`` is the union of the associated value types in ``A`` and ``B``
+
+All type checkers should accept code that would be accepted if these rules were followed, even if the type checker does not internally construct such a type.
+
+Notes:
+
+* For any TypedDict ``D`` with non-read-only items, the type constructed when merging two instances of type ``D`` will be equivalent to ``D``, and the type checker may reuse ``D`` internally.
```

Comment:
> and the type checker may reuse ``D`` internally

This is getting into implementation details of a type checker. Recommend deleting this phrase.